ApeWorX/ape
# fix: fix for docker issue `AttributeError: module 'rlp' has no attribute 'Serializable'`

### What I did
update dockerfile with the following:

- update ubuntu to 22.04
- update python to 3.9
- change install commands for ape
- change install commands for ape plugins
- add `RUN pip install eth-rlp==0.3.0`

fixes: #857 

### How I did it
I tested each of the changes and all were required to successfully build and run the docker image with ape. 

### How to verify it
fetch & checkout this repo/branch and `docker build .` then run the resulting hash with `docker run <hash>` and it will no longer fail with the error `AttributeError: module 'rlp' has no attribute 'Serializable'`

### Checklist
